README.md

```diff
@@ -52,48 +52,36 @@ Refer to our GitHub pages for the course materials and additional resources:
 - [Read](https://opensource.com/education/15/9/open-source-education-israel)
   an article by Laura Novich on [opensource.com](https://opensource.com)
 
-## Requirements
-
-Once we're in the ROSE directory, we need to verify we have pipenv
-installed.  In order to make sure we have pipenv installed:
-
-    pipenv --version
-
-If you don't have it installed, the best way is to install it only for
-your user:
-
-    python -m pip install --user pipenv
-
 ## Getting started
 
 The following commands should be performed only once; after creating the
 environment you will be connecting to the same environment each time you
 open a new session.
 
-Use pipenv to create a virtual environment and to install the rest of
+Use venv to create a virtual environment and to install the rest of
 the dependencies:
 
-    pipenv install
-
-You can also install development packages by running:
-
-    pipenv --dev install
+    python -m venv ./venv
```

Comment:
@nirs Basically the guide is meant for the students, and we're configuring their python to be the same as python3 for homework and class tasks, so using `python` instead of `python3` makes a difference for them conceptually.
What about adding a note instead of changing the code?

Reply:
Using `python` in the classes is a mistake, we should really use `python3`. They are 
not the same lanaguage - code that runs fine with one python version may not run with
another version.

If we already use python, we can keep using python in the readme for now, and change
it later everywhere to python3.